qa/rpc-tests/wallet_listnotes.py

```diff
@@ -88,19 +56,19 @@ def run_test(self):
         assert_equal(len(unspent_tx), 2)
         # sort low-to-high by amount (order of returned entries is not guaranteed)
         unspent_tx = sorted(unspent_tx, key=lambda k: k['amount'])
-        assert_equal(False,             unspent_tx[0]['change'])
```

Comment:
These reorderings seem trivial; did I do those? If so I don't think there was a good reason, and they should probably be reverted.